packages/block-editor/src/components/list-view/use-list-view-scroll-into-view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { getScrollContainer } from '@wordpress/dom';
+import { useLayoutEffect } from '@wordpress/element';
+
+export default function useListViewScrollIntoView( {
+	isSelected,
+	selectedClientIds,
+	rowItemRef,
+} ) {
+	const numBlocksSelected = selectedClientIds?.length;
+
+	useLayoutEffect( () => {
+		// Skip scrolling into view if more than one block is selected.
+		// This is to avoid scrolling the view in a multi selection where the user
+		// has intentionally selected multiple blocks within the list view,
+		// but the initially selected block may be out of view.
+		if (
+			! isSelected ||
+			numBlocksSelected !== 1 ||
+			! rowItemRef?.current ||
```

Comment:
`ref` should always be present, if it's not then I think we will have a bigger issue and we want to catch that. 😅 

```suggestion
			! rowItemRef.current ||
```